server/src/main/scala/drt/server/feeds/lgw/ResponseToArrivals.scala

```diff
@@ -0,0 +1,96 @@
+package drt.server.feeds.lgw
+
+import java.io.ByteArrayInputStream
+import drt.shared.Arrival
+import org.apache.commons.io.IOUtils
+import org.slf4j.{Logger, LoggerFactory}
+import scala.util.{Failure, Success, Try}
+import scala.xml.Node
+
+case class ResponseToArrivals(data: Array[Byte], locationOption: Option[String] ) {
+  val log: Logger = LoggerFactory.getLogger(getClass)
+
+  def getArrivals: List[(Arrival, Option[String])] = {
+    val is = new ByteArrayInputStream(data)
+    val xmlTry = Try(scala.xml.XML.load(is)).recoverWith {
+      case e: Throwable => log.error(s"Cannot load Gatwick XML from the response ${data.toString}", e); null
+    }
+    IOUtils.closeQuietly(is)
+    Try {
+      val xmlSeq = xmlTry.map(scala.xml.Utility.trimProper(_)).get
```

Comment:
I think using .get on an Option or a Try is bad practice, rather just flatMap over them, or do a match and handle the failure case.